File: Facade/Golang/VegRestaurant.go
package main

type VegRestaurant struct {
	Menu *Menu
}

func NewVegRestaurant() *VegRestaurant {
	return &VegRestaurant{
		Menu: &Menu{
			Name: "Veg Menu",
		},
	}
}

func (vegRestaurant *VegRestaurant) getMenus() *Menu {
	return vegRestaurant.Menu
}

File: Facade/PHP/VegRestaurant.php
<?php

require_once "IRestaurant.php";
require_once "Menu.php";

class VegRestaurant implements IRestaurant {
    private Menu $menu;

    public function __construct()
    {
        $this->menu = new Menu("Veg Menu");
    }

    public function getMenus() {
        return $this->menu;
    }
}